src/olympia/devhub/views.py

```diff
@@ -649,22 +649,16 @@ def handle_upload(
     submit=False,
     source=amo.UPLOAD_SOURCE_DEVHUB,
 ):
-    automated_signing = channel == amo.RELEASE_CHANNEL_UNLISTED
-
     user = request.user if request.user.is_authenticated else None
-    max_ip_length = FileUpload._meta.get_field('ip_address').max_length
     upload = FileUpload.from_post(
         filedata,
         filedata.name,
         filedata.size,
         addon=addon,
-        automated_signing=automated_signing,
-        ip_address=(core.get_remote_addr() or '')[:max_ip_length],
+        channel=channel,
         source=source,
         user=user,
     )
-    # The following log statement is used by foxsec-pipeline.
-    log.info('FileUpload created: %s' % upload.uuid.hex)
```

Comment:
moved up to `FileUpload.from_post` (the new addon submission api won't call devhub views)